src/nft/utils/chains.ts

```diff
@@ -0,0 +1,150 @@
+export type Chain = {
```

Comment:
This file looks like it duplicates existing funcitonality. It should be pulling the constants from existing files so that there is a single definition. You should _not_ be making a new ChainId enum - we already have too many of those, and it complicates typing because we need to coerce across types.